quickstart.md

```diff
@@ -0,0 +1,136 @@
+  - Then we need to "sign" the request, in the Playground we access it from the Ceramic instance. It's generated when you authenticate with 3ID-Connect.
+  - OUTPUT: ` `
+   - There will be nothing here because there's no profile attached to the DID.
+2. `await idx.set('basicProfile', { name: 'Your Name', description: 'Tell us about yourself!' emoji:'✌🏻' })`
+  - Notice that we did not provide our DID, but it's updating our record. This is because we default to the currently active ID.
+  - These fields are optional, but they can be set to be required by the schema.
+3. `await idx.merge('basicProfile', { emoji: '👍🏻'})`
+   - Updating records is easy, change any details you need to anytime you need to.
+
+## Creating a Custom Schema:
+Schema creation might seem daunting at first, but it's quite simple when you understand how it's all built. 
+
+### Helper Functions:
+To start we will need to create a few helper functions. These are all included in our `@ceramicstudio/idx-tools` package, this is just to demystify them. To use them you can just copy & paste the code into your browser console.
+1. Create Tile
+    ```javascript
+      /**
+      	* Create Tile is a simple function to create a TileDocument on the Ceramic Network.
+      	*/
+      const createTile = async (ceramic, content, metadata) => {
+      	if (ceramic.did == null) {
+          throw new Error('Ceramic instance is not authenticated')
+        }
+
+        if (metadata.controllers == null || metadata.controllers.length === 0) {
+          metadata.controllers = [ceramic.did.id] // if there's no controllers in the metadata, we assume that you're the only controller and automatically set it for you.
+        }
+
+      	// TileDocuments are what make up our Ceramic Network. 
+        const doc = await TileDocument.create(ceramic, content, metadata)
+        await ceramic.pin.add(doc.id) // Here we pin the stream to an IPFS Node.
+        
+      	return doc
+      }
+    ```
+2. Create Schema
+    ```javascript
+      const createSchema = async (ceramic, doc) => {
+        if(doc.id == null) {
+          return await createTile(ceramic, doc.content, {
+            controllers: doc.controllers,
+            schema: doc.schema ? docIDToString(doc.schema) : undefined
+          })
+        }
+
+        const loaded = await TileDocument.load(ceramic, doc.id)
+        console.log(loaded)
+        return loaded
+      }
+    ```
+3. Create Definition
+    ```javascript
+      /**
+      	* Definitions are a key part of how Ceramic categorizes data.
+      	* Without the definition the schema is inaccessible. 
+      	*/
+      const createDefinition = async (ceramic, definition) => {
+      	// We use our createTile helper to create our definition tile. 
+      	return await createTile(ceramic, 
+      		definition, 
+      		{
+      			schema: 'ceramic://k3y52l7qbv1fry1fp4s0nwdarh0vahusarpposgevy0pemiykymd2ord6swtharcw'
```

Comment:
Might be safer to import the definition schema URL from `idx-constants` here, no?